lib/locales/hu.ts

```diff
@@ -0,0 +1,938 @@
+    francis_diaz_del_rincon_priest_and_companions_martyrs: 'Saint Francis Díaz del Rincon, Priest, and Companions, Martyrs',
+    francis_ferdinand_de_capillas_priest: 'Saint Francis Ferdinand de Capillas, Priest and Martyr',*/
+    francis_of_assisi: 'Assisi Szent Ferenc',
+    francis_of_assisi_patron_of_italy: 'Assisi Szent Ferenc, Olaszország patrónusa',
+    /*francis_of_paola_hermit: 'Saint Francis of Paola, Hermit',
+    francis_solanus_priest: 'Saint Francis Solanus, Priest',
+    francis_xavier_priest: 'Saint Francis Xavier, Priest',
+    francis_xavier_seelos_priest: 'Blessed Francis Xavier Seelos, priest',
+    francois_de_montmorency_laval_bishop: 'Saint François de Laval, Bishop',
+    frederic_janssoone_priest: 'Blessed Frédéric Janssoone, Priest',
+    frederic_ozanam_founder: 'Blessed Frédéric Ozanam, Founder',
+    friday_of_the_passion_of_the_lord: 'Friday of the Passion of the Lord',
+    fridolin_of_sackingen_monk: 'Saint Fridolin of Säckingen, Monk',
+    fructuosus_of_braga_martin_of_braga_and_gerald_of_braga_bishops: 'Saints Fructuosus, Martin and Gerald of Braga, Bishops',
+    fructuosus_of_tarragona_bishop_and_augurius_of_tarragona_and_eulogius_of_tarragona_deacons_martyrs: 'Saints Fructuosus, Bishop, Augurius and Eulogius, Deacons, Martyrs',
+    fursa_of_peronne_abbot: 'Saint Fursa, Abbot and Missionary',
+    gabriel_taurin_dufresse_bishop: 'Saint Gabriel-Taurin Dufresse, Bishop and Martyr',
+    gall_of_switzerland_abbot: 'Saint Gall, Abbot and Missionary',
+    genevieve_of_paris_virgin: 'Saint Genevieve, Virgin',
+    george_matulaitis_bishop: 'Blessed George Matulaitis, Bishop',
+    george_of_lydda_martyr: 'Saint George, Martyr',
+    george_of_lydda_martyr_patron_of_england: 'Saint George, Martyr, Patron of England',
+    george_preca_priest: 'Saint George Preca, Priest',
+    gerard_of_csanad_bishop: 'Saint Gerard of Csanád, Bishop and Martyr',
+    germain_of_paris_bishop: 'Saint Germain of Paris, Bishop',
+    germanus_of_auxerre_bishop: 'Saint Germanus of Auxerre, Bishop',
+    gertrude_of_nivelles_abbess: 'Saint Gertrude of Nivelles, Abbess',
+    gertrude_the_great_virgin: 'Saint Gertrude, Virgin',*/
+    gisela_of_hungary: 'Boldog Gizella',
+    /*gobnait_of_ballyvourney_virgin: 'Saint Gobnait, Virgin',
+    gorazd_of_moravia_and_companions: 'Saint Gorazd and Companions',
+    gotthard_of_hildesheim_bishop: 'Saint Gotthard, Bishop',
+    gratia_of_cattaro_religious: 'Blessed Gratia of Cattaro, Religious',
+    gregory_grassi_francis_fogolla_and_anthony_fantosati_bishops_and_companions_martyrs: 'Saints Gregory Grassi, Francis Fogolla and Anthony Fantosati, Bishops, and Companions, Martyrs',
+    gregory_i_the_great_pope: 'Saint Gregory the Great, Pope and Doctor of the Church',
+    gregory_of_narek_abbot: 'Saint Gregory of Narek, Abbot and Doctor of the Church',
+    gregory_vii_pope: 'Saint Gregory VII, Pope',*/
+    guardian_angel_of_portugal: 'Portugália őrangyala',
```

Comment:
I am not sure if this is correct. According to [_Direktórium 2022_](http://tar.liturgia.hu/Naptar/DIR2022_JAV_2021-12-13.pdf#page=161), `holy_guardian_angels` are translated as `Szent Őrzőangyalok`, therefore I would translated this as `Portugália Őrzőangyala`, however, I might be wrong (I might be not aware of some Hungarian grammar rules), because when I searched the web, all I could find is `Portugália őrangyala`: I could find [39 results of `Portugália Őrzőangyala`](https://www.google.com/search?client=firefox-b-d&q=%22Portug%C3%A1lia%2B%C5%90rz%C5%91angyala%22+site%3A.hu) and [71 results of `Portugália Őrangyala`](https://www.google.com/search?client=firefox-b-d&q=%22Portug%C3%A1lia%2B%C5%90rangyala%22+site%3A.hu).

Reply:
Well, I have no idea which is right.

Reply:
What is correct in Hungarian in general: _őrzőangyal_ or _őrangyal_?

Reply:
Both are correct.

Reply:
If both are correct, then I suggest to use `Őrzőangyal` in order to be consistent with the value of `holy_guardian_angels`.

Reply:
Ok.